creator-node/src/components/replicaSet/replicaSetController.js

```diff
@@ -59,6 +60,17 @@ const respondToURSMRequestForProposalController = async (req) => {
  */
 const syncRouteController = async (req, res) => {
   const serviceRegistry = req.app.get('serviceRegistry')
+  req.logger.debug(`/sync serviceRegistry empty: ${_.isEmpty(serviceRegistry)}`)
+  req.logger.debug(
+    `/sync serviceRegistry?.nodeConfig empty: ${_.isEmpty(
+      serviceRegistry?.nodeConfig
+    )}`
+  )
+  req.logger.debug(
+    `/sync serviceRegistry?.syncQueue empty: ${_.isEmpty(
+      serviceRegistry?.syncQueue
+    )}`
+  )
```

Comment:
you want to keep all this?

Reply:
I figured it's good to keep for debugging on stage. there's an error still happening sometimes where `serviceRegistry.syncQueue` is null or undefined from some reason